build.rs

```diff
@@ -7,23 +7,38 @@ use std::env;
 use std::path::PathBuf;
 
 fn main() {
-	let mut c = Build::new();
-
-	c.warnings(false);
-	c.flag("-std=c99");
-	c.file("src/c/YGEnums.c");
-	c.file("src/c/YGNodeList.c");
-	c.file("src/c/Yoga.c");
-	c.compile("libyoga.a");
+	Build::new()
+		.cpp(true)
+		// https://github.com/facebook/yoga/blob/c5f826de8306e5fbe5963f944c75add827e096c3/BUCK#L13
+		.flag("-std=c++1y")
+		// https://github.com/facebook/yoga/blob/c5f826de8306e5fbe5963f944c75add827e096c3/yoga_defs.bzl#L49-L56
+		.flag("-fno-omit-frame-pointer")
+		.flag("-fexceptions")
+		.flag("-Wall")
+		.flag("-O3")
+		.flag("-ffast-math")
+		// https://github.com/facebook/yoga/blob/c5f826de8306e5fbe5963f944c75add827e096c3/yoga_defs.bzl#L58-L60
+		.flag("-fPIC")
+		// C++ Files
+		.file("src/c/Utils.cpp")
+		.file("src/c/YGConfig.cpp")
+		.file("src/c/YGEnums.cpp")
+		.file("src/c/YGFloatOptional.cpp")
+		.file("src/c/YGLayout.cpp")
+		.file("src/c/YGNode.cpp")
+		.file("src/c/YGNodePrint.cpp")
+		.file("src/c/YGStyle.cpp")
+		.file("src/c/Yoga.cpp")
+		.compile("libyoga.a");
 
 	let bindings = bindgen::Builder::default()
 		.rust_target(RustTarget::Stable_1_21)
-		.blacklist_type("max_align_t") // This fails `cargo test` so disable for now
-		.blacklist_type("FP_INFINITE")
-		.blacklist_type("FP_NAN")
-		.blacklist_type("FP_NORMAL")
-		.blacklist_type("FP_SUBNORMAL")
-		.blacklist_type("FP_ZERO")
+		.no_convert_floats()
+		// .enable_cxx_namespaces()
```

Comment:
@victorporof I couldn't get the code building with this. I'm familiar with C++ namespaces but I'm not sure what it means to "enable" or "disable" them.